source/includes/indexes/indexes.php

```diff
@@ -30,3 +30,14 @@
 $document = $collection->findOne(['title' => 'Sweethearts']);
 echo json_encode($document) , PHP_EOL;
 // end-index-single-query
+
+// start-multikey
+$indexName = $collection->createIndex(['cast' => 1]);
+// end-multikey
+
+// start-index-array-query
+$document = $collection->findOne(
+    ['cast' => ['$in' => ['Aamir Khan', 'Kajol']]]
+);
+echo json_encode($document) , PHP_EOL;
```

Comment:
In the index landing PR (https://github.com/mongodb/docs-php-library/pull/127), I suggested using some driver functions to print the value as extended JSON. It looks like that was missed in a subsequent PR for single field indexes (https://github.com/mongodb/docs-php-library/pull/134), upon which I expect this was based.

I'll defer to you if it's worth changing. The expected output in the RST doesn't show any special BSON types, so it's probably of little consequence.

```suggestion
echo json_encode($document), PHP_EOL;
```

No need for preceding whitespace for a comma. It would be appropriate for a concatenation operator (dot), though.

Reply:
I think it's ok to use the simpler `json_encode()` method in these contexts. I will change the whitespace though!